### PR TITLE
change: [M3-7684] - Keep Create Volumes Page Error Notification Position Consistent

### PR DESCRIPTION
**Pull request by pmakode-akamai**
## Description 📝

Keeping the Create Volumes Page error notification position consistent with other tabs like NodeBalancers.

## Changes  🔄

- For restricted users: 
    - The position of error notification has been moved from inside the box to outside, which is now consistent with other tabs like NodeBalancers.
    - Used `getRestrictedResourceText` utility to show a Notice / error notification message.

## Target release date 🗓️
July 8th, 2024

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-07-01 at 11 21 53 PM | Screenshot 2024-07-01 at 11 48 30 PM |

### Prerequisites
(How to setup test environment)
- Log in with a restricted user account (read-only access) and go to the Create Volumes Page via volumes/create route directly from you browser.

### Reproduction steps
 -  Observe as restricted user, error notification shows out of the box (not inside as before) and you cannot create Volumes.

### Verification steps
- As a restricted user, compare Create Volumes Page (route: volumes/create) with Create NodeBalancers Page (route: nodebalancers/create) to ensure consistency.   

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by mjac0bs — COMMENTED** [review]
Thanks Purvesh! I confirmed the permissions error displays as described for a restricted user and left a comment about the other error messages.

image


Two small notes:

- Can you add a changeset for this PR? (`yarn changeset`) and describe the change? This will get incorporated into our changelog for our next release.

- I just updated the target release date in this PR's description. We release every other week on Mondays, which makes our next release on Monday, July 8th. (If you're looking for our release schedule in our internal docs and haven't found it, I can send it your way.)

File: packages/manager/src/features/Volumes/VolumeCreate.tsx [hunk 1]
```diff
@@ -254,6 +255,18 @@ export const VolumeCreate = () => {
         }}
         title="Create"
       />
+      {error && <Notice spacingTop={8} text={error} variant="error" />}
```
[the remark this answers is not in the document]
Comment (pmakode-akamai) [reply packages/manager/src/features/Volumes/VolumeCreate.tsx:258]: Hi @mjac0bs, could you please check [here](https://github.com/pmakode-akamai/manager/blob/aae1a377177803e69498b15c6d48b0e06d89e0b4/packages/manager/src/features/NodeBalancers/NodeBalancerCreate.tsx#L482) why does the general error message is kept outside in the case of NodeBalancers?  Shouldn't they be consistent?

[the remark this answers is not in the document]
Comment (bnussman-akamai) [reply packages/manager/src/features/Volumes/VolumeCreate.tsx:258]: I don't remember exactly where we landed, but I'd agree that the permission error belongs outside the form and the "general error" belongs inside the paper on a small form like this one.

[the remark this answers is not in the document]
Comment (pmakode-akamai) [reply packages/manager/src/features/Volumes/VolumeCreate.tsx]: Ok. Keeping permission error outside the form and "general error" inside the paper.

File: packages/manager/.changeset/pr-10632-changed-1719914806428.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Changed
+---
+
+Position of error notif has been moved from inside the box to outside & used getRestrictedResourceText utility for the Notice ([#10632](https://github.com/linode/manager/pull/10632))
```
Comment (mjac0bs) [inline packages/manager/.changeset/pr-10632-changed-1719914806428.md:5]:
```suggestion
Use `getRestrictedResourceText` utility and move restrictions Notice to top of Volume Create ([#10632](https://github.com/linode/manager/pull/10632))
```

Nit: a slightly more brief version

**Review by mjac0bs — APPROVED** [review]

File: packages/manager/.changeset/pr-10632-changed-1719914806428.md (hunk 2 above)
Comment (pmakode-akamai) [reply packages/manager/.changeset/pr-10632-changed-1719914806428.md]: Done!

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**